### PR TITLE
Script for debugging a possible drop DB error

### PR DESCRIPTION
**Pull request by xBis7**
context: https://github.com/G-Research/gr-oss/issues/720#issuecomment-2230898954

The script is using methods for spark tests 3 and 4 instead of the actual tests, for two reasons
* It's easier to see the steps
* It's easier to avoid setting up policies every time without adding unecessary parameters in the existing tests

Example of running the script
```
./big-data-c3-tests/trino-spark-tests/debugging_scripts/test_spark_drop_error.sh /home/projects true 5 showDbsAfter
```

### CODE REVIEW

File: trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/debugging_scripts/test_spark_drop_error.sh [hunk 1]
```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
```
Comment (xBis7) [inline trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/debugging_scripts/test_spark_drop_error.sh:1]: @GeorgeJahad Maybe rename the script to `debug_spark_drop_db_error.sh`?

Comment (GeorgeJahad) [reply trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/debugging_scripts/test_spark_drop_error.sh:1]: I would call it debug_no_such_namespace_error.sh

**Review by GeorgeJahad — APPROVED** [review]
This is good enough for now.  Longer term we may want to create more generic scripts and then build tools on top of that, but that isn't important enough yet.

thanks!

File: trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/debugging_scripts/test_spark_drop_error.sh (hunk 1 above)
Comment (xBis7) [reply trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/debugging_scripts/test_spark_drop_error.sh:1]: Done.

**Pull request merged**